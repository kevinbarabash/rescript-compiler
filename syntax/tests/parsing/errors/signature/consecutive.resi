let x: int let y: float

export z1: int export z2: float
