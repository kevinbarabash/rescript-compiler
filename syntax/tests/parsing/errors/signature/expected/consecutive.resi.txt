
  Syntax error!
  ./tests/parsing/errors/signature/consecutive.resi:1:11-14

  1 │ let x: int let y: float
  2 │ 
  3 │ export z1: int export z2: float

  consecutive specifications on a line must be separated by ';' or a newline


  Syntax error!
  ./tests/parsing/errors/signature/consecutive.resi:3:15-21

  1 │ let x: int let y: float
  2 │ 
  3 │ export z1: int export z2: float
  4 │ 

  consecutive specifications on a line must be separated by ';' or a newline

val x : int
val y : float
val z1 : int[@@genType ]
val z2 : float[@@genType ]